.github/workflows/promote_unstable_docs.yml

```
@@ -36,3 +36,23 @@ jobs:
           RDME_API_KEY: ${{ secrets.README_API_KEY }}
         run: |
           python ./.github/utils/promote_unstable_docs.py --version ${{ steps.version.outputs.version }}
+
+      - name: Promote unstable docs for Docusaurus
+        run: |
+          git checkout main
+          python ./.github/utils/promote_unstable_docs_docusaurus.py --version ${{ steps.version.outputs.version }}
+
+      - name: Create Pull Request with Docusaurus docs updates
+        uses: peter-evans/create-pull-request@v7
+        with:
+          token: ${{ secrets.HAYSTACK_BOT_TOKEN }}
+          commit-message: "Promote unstable docs for Haystack ${{ steps.version.outputs.version }}"
+          branch: promote-unstable-docs-${{ steps.version.outputs.version }}
+          base: main
+          title: "docs: promote unstable docs for Haystack ${{ steps.version.outputs.version }}"
+          add-paths: |
+            docs-website
+          body: |
+            This PR promotes the unstable docs for Haystack ${{ steps.version.outputs.version }} to stable.
+            It is expected to run at the time of the release.
+            You can inspect the docs preview and merge it.
```

Just a clarification for the reviewer on what to check.
```suggestion
            You can inspect the docs preview and merge it. There should now be only one unstable version representing the next (main) branch.
```